src/components/drawing-widget/drawing-widget.component.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import ReactImageAnnotate from "react-image-annotate";
+
+interface DrawingWidgetProps {
+  selectedImage: string;
+  taskDescription: string;
+  images: { src: string; name: string }[];
+  regionClsList?: string[];
+  enabledTools?: string[];
+}
+
+const DrawingWidget: React.FC<DrawingWidgetProps> = () => {
+  return (
+    <div className="drawing-widget">
+      <ReactImageAnnotate
+        labelImages
+        regionClsList={["Alpha", "Beta", "Charlie", "Delta"]}
+        regionTagList={["tag1", "tag2", "tag3"]}
+        images={[
+          {
+            src: "https://placekitten.com/408/287",
+            name: "Cat Physical Exam",
+            regions: [],
+          },
```

Comment:
You probably don't want to persist this to source control but if so, I would add a TODO that bounds me to remove this later.